ckan/lib/navl/validators.py

```diff
@@ -74,6 +74,8 @@ def ignore(key, data, errors, context):
     raise StopOnError
 
 def default(default_value):
+    '''When key is missing or value is 'empty', set it with a default value.
+    See the code for how 'empty' is defined.'''
```

Comment:
I don't think it hurts to repeat ourselves here.

`When key is missing or value is an empty string or None, replace it with a default value`

One day we could include the validator docstrings in the proper docs.